scraper.rb

```diff
@@ -40,6 +82,89 @@ class Governor < Scraped::HTML
   field :party do
     noko.css('td')[4].text.tidy
   end
+
+  # Get extra fields from the Pombola data, if present:
+
+  field :email do
+    pombola_contact_details_of_type('email')
+  end
+
+  field :phone do
+    pombola_contact_details_of_type('phone')
+  end
+
+  field :twitter do
+    pombola_contact_details_of_type('twitter')
+  end
+
+  field :facebook_url do
+    pombola_contact_details_of_type('facebook')
+  end
+
+  field :birth_date do
+    pombola_data['birth_date']
+  end
+
+  field :gender do
+    pombola_data['gender']
+  end
+
+  field :identifier__shineyoureye do
+    identifiers = pombola_data.fetch('identifiers', [])
+    identifier = identifiers.find { |i| i['scheme'] == 'pombola-slug' }
+    identifier ? identifier['identifier'] : ''
+  end
+
+  field :image_url do
+    image_urls = pombola_data.fetch('images', []).map { |i| i['url'] }
+    image_urls.first || ''
+  end
+
+  field :website_url do
+    pombola_contact_details_of_type('website')
+  end
+
+  field :other_names do
+    pombola_name = pombola_data.fetch('name', '').tidy
+    if pombola_name.downcase == name.downcase
+      ''
+    else
+      pombola_name
+    end
+  end
+
+  private
+
+  def id_to_pombola_data
+    @@id_to_pombola_data ||= begin
+      url = 'http://www.shineyoureye.org/media_root/popolo_json/persons.json'
+      popolo = JSON.parse(open(url, &:read))
+      pombola_id_to_id = ID_TO_POMBOLA_ID.collect do |t|
+        ["core_person:#{t[1]}", t[0]]
+      end.to_h
```

Comment:
Also, given map and collect are the same thing, at least AIUI, why collect here and map later in the block? (this may be my ignorance of ruby idiom showing)